### ISSUE TITLE
Input delay after FocusGained (switching to NeoVim)

### ISSUE THREAD

**Issue by TamaMcGlinn** [gbprod/yanky.nvim#186]
I recently started having a problem on a machine I access over SSH where, as I switch focus back to NeoVim, input is delayed by about 3 seconds, and the cursor disappears for a few seconds longer. 

After bisecting through my plugins, I found that removing yanky fixed the issue. I then tried many commits of yanky, back to v1.0.0 but all have the same result.

However, this problem does not occur on another machine I use locally (not over SSH), even though I'm on the same latest commit of Yanky.

After a clue elsewhere about FocusGained aucmd, I found that the problem is with [the sync_with_ring option](https://github.com/gbprod/yanky.nvim?tab=readme-ov-file#system_clipboardsync_with_ring).

Long story short, if you are running into this, set sync_with_ring to `false`. It may be a good idea to set this as the default.

If you would like further testing on the machine with the problem, let me know and I'll be happy to help.

**Comment by gbprod** [comment]
Can you share your config ?
This is probably because you're using sqlite as storage. try shada instead  ;)
